addon/-private/system/model/internal-model.js

```diff
@@ -435,6 +435,7 @@ InternalModel.prototype = {
     this.record._notifyProperties(dirtyKeys);
 
   },
+
```

Comment:
Actually, all other methods in this file have whitespace separators, so I left it in as a small style fix.